### PR TITLE
Unify conversation layout

### PR DESCRIPTION
**Pull request by daehn**
# What's in this PR?

* Images are no longer edge to edge and follow the same rules as other messages.
* The same applies for video messages and our old style attachments (YouTube and Vimeo).

### CODE REVIEW

**Comment by typfel** [comment]
No round corners on images?

**Comment by daehn** [comment]
@typfel no, so far not.

**Comment by typfel** [comment]
@daehn ok, but the file cells preview cell should have rounded corner?

File: Wire-iOS/Resources/Classy/stylesheet.cas [hunk 1]
```diff
@@ -602,6 +602,18 @@ ToolTipViewController {
     }
 }
 
+/* MediaPreviewViewController */
+
+MediaPreviewViewController {
```
Comment (mikeger) [inline Wire-iOS/Resources/Classy/stylesheet.cas]: The class does not seem to have a property of `minimumDismissMagnitude`. Wrong selector?

Comment (daehn) [inline Wire-iOS/Resources/Classy/stylesheet.cas]: Yes, should be viewHeight as well.

**Comment by typfel** [comment]
@daehn you are adding new snapshots because the cells changed size. Shouldn't we delete some old reference snapshots then?

**Comment by daehn** [comment]
@typfel @mikeger updated!

**Comment by daehn** [comment]
@typfel @mikeger  🚀

**Pull request merged**